string agent_id
bool registered

#    bool registered
#        true
